src/app/clusters/ota-requestor/OTARequestor.cpp

```diff
@@ -809,9 +815,22 @@ void OTARequestor::StoreCurrentUpdateInfo()
     }
 }
 
+void OTARequestor::Reset()
+{
+    mProviderLocation.ClearValue();
+    mUpdateToken.reduce_size(0);
+    RecordNewUpdateState(OTAUpdateStateEnum::kIdle, OTAChangeReasonEnum::kSuccess);
+    mTargetVersion = 0;
+
+    // Persist in case of a reboot or crash
+    StoreCurrentUpdateInfo();
+}
+
 // Invoked when the device becomes commissioned
 void OTARequestor::OnCommissioningCompleteRequestor(const DeviceLayer::ChipDeviceEvent * event, intptr_t arg)
 {
+    ChipLogDetail(SoftwareUpdate, "//is: OTARequestor::OnCommissioningCompleteRequestor");
```

Comment:
I know you're already aware of these debug statements lingering but just putting this here so it's not forgotten

Reply:
Yea these lines are already deleted in VSCode and shows a red triangle symbol saying so, but they're still showing up on my git branch even with a force re-commit.

Reply:
Poked around VScode and was able to fix it.